pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -59,10 +71,19 @@ static_resources:
           - {{ .MixerSubjectAltName }}
 {{- end }}
     type: STRICT_DNS
+{{- if .IPv6Proxy }}
+    dns_lookup_family: AUTO
```

Comment:
Do we want auto here or V6_Only?

Reply:
AUTO would work as I am using it right now, V6_ONLY is more strict and will break when dual stack support will be supported.  I do not have strong opinion here though.